app/models.py

```diff
@@ -85,12 +87,46 @@ class Config:
         }
 
 
+class QuestionResponse(Question):
+    """Model for the response of any request that returns a question"""
+
+    question_set_id: str
```

Comment:
when we want the backend to return a question, why are we just returning the question_set_id?